eventlet/websocket.py

```diff
@@ -475,12 +475,8 @@ def close(self):
             self._send_closing_frame(True)
             self.socket.shutdown(True)
         except SocketError as e:
-            if e.errno == errno.ENOTCONN:
-                pass  # already disconnected just close socket
-            else:
-                self.log.write('Error on socket shutdown {0}:{1}: {2}'.format(
-                    self.client_host, self.client_port, e
-                ))
+            if e.errno != errno.ENOTCONN:
+                self.log.write('Error on socket shutdown {0}: {1}'.format(self.client_peer, e))
```

Comment:
server hostname and websocket url could be useful for reader.

General idea is - log should contain as much as possible information that could help to debug a problem.